lib/spack/spack/build_systems/autotools.py

```diff
@@ -151,13 +151,25 @@ def configure(self, spec, prefix):
         options = ['--prefix={0}'.format(prefix)] + self.configure_args()
         inspect.getmodule(self).configure(*options)
 
+    def build_targets(self):
```

Comment:
I did that at first, but changed it to match `configure_args`, which is also a member function. I agree though, I would also change `configure_args` to be a normal attribute.